homu/main.py

```diff
@@ -1007,11 +1003,18 @@ def queue_handler():
     os.environ['GIT_SSH'] = os.path.join(os.path.dirname(__file__), 'git_helper.py')
     os.environ['GIT_EDITOR'] = 'cat'
 
+    tmp_ssh_key = None
+    if git_cfg['local_git']:
+        tmp_ssh_key = tempfile.NamedTemporaryFile(prefix='homu-sshkey')
```

Comment:
This commit was intentionally tying the tmpfile lifecycle to Python GC, so that it goes away if the process exits etc.  That's why I had to do the hack of passing it as an argument to the "mainloop" just so it was kept alive.